pkg/api/controller.go

```diff
@@ -620,13 +625,16 @@ func (c *Controller) Shutdown() {
 
 func (c *Controller) StartBackgroundTasks(reloadCtx context.Context) {
 	taskScheduler := scheduler.NewScheduler(c.Log)
-	taskScheduler.RunScheduler(reloadCtx)
+	taskScheduler.RunScheduler(reloadCtx, scheduler.RateLimit)
 
 	// Enable running garbage-collect periodically for DefaultStore
 	if c.Config.Storage.GC && c.Config.Storage.GCInterval != 0 {
 		c.StoreController.DefaultStore.RunGCPeriodically(c.Config.Storage.GCInterval, taskScheduler)
 	}
 
+	// Enable running rebuild dedupe index
+	c.StoreController.DefaultStore.RunRebuildDedupeIndex(time.Duration(0), taskScheduler, c.Config.Storage.Dedupe)
```

Comment:
Are these durations correct? In all RunRebuildDedupeIndex in this file.

Reply:
time.Duration(0) means that scheduler is not periodic, just one time thing.